src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -755,7 +763,23 @@ class AudiusAPIClient {
       throw new Error('AudiusAPIClient must be initialized before use')
   }
 
-  async _getResponse<T>(resource: string): Promise<Nullable<T>> {
+  async _getResponse<T>(
+    path: string,
+    params: QueryParams = {}
+  ): Promise<Nullable<T>> {
+    if (this.initializationState.state !== 'initialized')
+      throw new Error('_constructURL called uninitialized')
+
+    if (this.initializationState.type === 'libs' && window.audiusLibs) {
+      const data = await window.audiusLibs.discoveryProvider._makeRequest({
+        endpoint: this._formatPath(path),
+        queryParams: params
+      })
+      // TODO: Type boundaries of API
+      return { data } as any
```

Comment:
If it does fail (after it tries a bunch of things), it returns null :)

{ data } as T doesn't work unf. i tried for a while to get it to, but 

```
Conversion of type '{ data: any; }' to type 'T' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  'T' could be instantiated with an arbitrary type which could be unrelated to '{ data: any; }'
```

Reply:
I think it's b/c data any implicitly